x-pack/plugin/ilm/qa/rest/src/yamlRestTest/resources/rest-api-spec/test/ilm/10_basic.yml

```
@@ -58,6 +58,96 @@ setup:
       ilm.get_lifecycle:
         policy: "my_timeseries_lifecycle"
 
+---
+"Test Policy Get":
+  - do:
+      ilm.put_lifecycle:
+        policy: "my_timeseries_lifecycle_1"
+        body: |
+          {
+            "policy": {
+              "phases": {
+                "warm": {
+                  "min_age": "10s",
+                  "actions": {
+                    "forcemerge": {
+                      "max_num_segments": 10000
+                    }
+                  }
+                }
+              }
+            }
+          }
+
+  - do:
+      ilm.put_lifecycle:
+        policy: "my_timeseries_lifecycle_2"
+        body: |
+          {
+            "policy": {
+              "phases": {
+                 "delete": {
+                   "min_age": "30s",
+                   "actions": {
+                     "delete": {}
+                   }
+                 }
+              }
+            }
+          }
+
+  - do:
+      ilm.put_lifecycle:
+        policy: "other_lifecycle"
+        body: |
+          {
+            "policy": {
+              "phases": {
+                "warm": {
+                  "min_age": "10s",
+                  "actions": {
+                    "forcemerge": {
+                      "max_num_segments": 10000
+                    }
+                  }
+                },
+                "delete": {
+                  "min_age": "30s",
+                  "actions": {
+                    "delete": {}
+                  }
+                }
+              }
+            }
+          }
+
+  - do:
+      ilm.get_lifecycle:
+        policy: "my_timeseries*"
+  - match: { my_timeseries_lifecycle_1.version: 1 }
+  - match: { my_timeseries_lifecycle_2.version: 1 }
+
+  - do:
+      ilm.get_lifecycle:
+        policy: "my_timeseries_lifecycle_1, my_timeseries_lifecycle_2"
```

I believe this extra space is causing test failures:
```suggestion
        policy: "my_timeseries_lifecycle_1,my_timeseries_lifecycle_2"
```